packages/cli-e2e/docker/config/config.yaml

```diff
@@ -5,15 +5,27 @@
 #       https://github.com/verdaccio/verdaccio/tree/master/conf
 
 # path to a directory with all packages
+# TODO CDX-727: Sort out the verdaccio config shenanigans
+# Use this one for local tests:
+# storage: /verdaccio/storage/data
+# Use this one for CI (leave uncommented in PR)
 storage: ./storage/data
 # path to a directory with plugins to include
+# TODO CDX-727: Sort out the verdaccio config shenanigans
+# Use this one for local tests:
+# plugins: /verdaccio/plugins
+# Use this one for CI (leave uncommented in PR)
 plugins: ./plugins
 web:
   enable: false
   title: Verdaccio
 
 auth:
   htpasswd:
+    # TODO CDX-727: Sort out the verdaccio config shenanigans
+    # Use this one for local tests:
+    # plugins: /verdaccio/storage/htpasswd
+    # Use this one for CI (leave uncommented in PR)
```

Comment:
Ideally, I want the CI and CD to be compat on their path, but I don't have the time nor the will to work on that right now.
It's better than it is currently (i.e. all brokey for local), so I consider the boyscout principle to be (lazily) satisfied, period 😂 .